Properties/AssemblyInfo.cs

```diff
@@ -1,16 +0,0 @@
-using System.Reflection;
-using System.Runtime.InteropServices;
-
-[assembly: AssemblyTitle("NFive Core SDK")]
-[assembly: AssemblyDescription("NFive core SDK for plugin development")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("NFive")]
-[assembly: AssemblyProduct("NFive SDK")]
-[assembly: AssemblyCopyright("Copyright © NFive 2018-2021")]
```

Comment:
Need to move the relevant fields here to the csproj file as these are used in `NFive.SDK.Core.nuspec`, [reference here](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generateassemblyinfo),